src/ert/_c_wrappers/enkf/subst_config.py

```diff
@@ -160,6 +160,8 @@ def __repr__(self):
         return f"SubstConfig({str(self)})"
 
     def __str__(self):
+        if not self._address():
+            return ""
```

Comment:
Why not `return "<SubstConfig()>"`?

Reply:
i'm just building on top of the existing pattern given by repr calling str - i guess the focus when that design was conceived was on repr, rather than string. does that make sense? or is the empty output of `__str__` in case of uninitialized config unacceptable? then i could try to rewrite / change the pattern a bit